support-frontend/assets/pages/supporter-plus-landing/components/paymentTsAndCs.test.tsx

```diff
@@ -5,22 +5,22 @@ import { PaymentTsAndCs } from './paymentTsAndCs';
 
 describe('Payment Ts&Cs Snapshot comparison', () => {
 	const paymentProductKeys = [
-		['Contribution', 'UnitedStates'],
-		['SupporterPlus', 'GBPCountries'],
-		['TierThree', 'GBPCountries'],
-		['OneTimeContribution', 'AUDCountries'],
-		['GuardianAdLite', 'GBPCountries'],
-		['DigitalSubscription', 'GBPCountries'],
+		['Contribution', 'UnitedStates', 0],
+		['SupporterPlus', 'GBPCountries', 12],
+		['TierThree', 'GBPCountries', 27],
+		['OneTimeContribution', 'AUDCountries', 0],
+		['GuardianAdLite', 'GBPCountries', 0],
+		['DigitalSubscription', 'GBPCountries', 0],
 	];
 	it.each(paymentProductKeys)(
 		`paymentTs&Cs render product %s for region %s correctly`,
-		(paymentProductKey, countryGroupId) => {
+		(paymentProductKey, countryGroupId, amount) => {
 			const { container } = render(
 				<PaymentTsAndCs
 					contributionType={'MONTHLY'}
```

Comment:
Would it be worthwhile testing any alternative `contributionType` values that the `PaymentTsAndCs` contains conditional checks on?

Reply:
Yes (like `orderSummaryTsAndCs.test.tsx`) have expanded to include contributionType = MONTHLY and ANNUAL. Also added a PromoCode test. Applied to stories and tests. Updated